base/applications/regedit/main.c

```diff
@@ -191,7 +191,11 @@ BOOL TranslateChildTabMessage(PMSG msg)
 
     if (msg->wParam != VK_TAB) return FALSE;
     if (GetParent(msg->hwnd) != g_pChildWnd->hWnd) return FALSE;
+#ifdef __REACTOS__
```

Comment:
This file is not in sync as well.

https://github.com/reactos/reactos/blob/7cc89e520c3ac4182e9f6db13610e01a5e54efe5/media/doc/WINESYNC.txt#L322